src/widgets/ActivityDetails/model/itemAnswer.ts

```diff
@@ -1,6 +1,7 @@
-import { AnswerTypesPayload } from "~/shared/api"
+import { AlertDTO, AnswerTypesPayload } from "~/shared/api"
 
 export type ItemAnswer = {
   answer: AnswerTypesPayload | null
   itemId: string
+  alert: Array<AlertDTO>
```

Comment:
Also, we should separate responsibilities, so the answers' logic should not include the alerts' logic

Reply:
I believe we should use DTO here because it is the last step before sending responses. Here critical to converting into DTO because we have a direct dependency on DTO contract

Reply:
now I see